core/src/executor/evaluate/mod.rs

```diff
@@ -311,10 +311,12 @@ async fn evaluate_function<'a, 'b: 'a, 'c: 'a, T: GStore>(
             if let Some(custom_func) = custom_func {
                 let args: Vec<Evaluated<'_>> = stream::iter(exprs).then(eval).try_collect().await?;
 
-                let args = args
-                    .into_iter()
-                    .map(|expr| Value::try_from(expr).unwrap())
-                    .collect::<Vec<_>>();
+                let fargs: Vec<Evaluated<'_>> = if let Some(fargs) = &custom_func.args {
+                    let fargs: Vec<Expr> = fargs.iter().filter_map(|y| y.default.clone()).collect();
+                    stream::iter(&fargs).then(eval).try_collect().await?
```

Comment:
@panarch , please can ou help me to debug this part ?
i got this error `returns a value referencing data owned by the current function`